packages/esm-patient-labs-app/src/lab-orders/add-lab-order/test-type-search.component.tsx

```diff
@@ -122,8 +122,13 @@ function TestTypeSearchResults({ searchTerm, openOrderForm, focusAndClearSearchI
             </div>
           )}
           <div className={styles.resultsContainer}>
-            {filteredTestTypes.map((testType, index) => (
-              <TestTypeSearchResultItem key={index} openOrderForm={openOrderForm} t={t} testType={testType} />
+            {filteredTestTypes.map((testType) => (
+              <TestTypeSearchResultItem
+                key={testType.conceptUuid}
```

Comment:
Now that we're reasonably certain that the conceptUuids are unique, we can use them as keys in the test type search results list.